js/language_server/fixtures/test.py

```diff
@@ -0,0 +1,2 @@
+if 5 == 5:
```

Comment:
fixtures?

Reply:
should probably rename to targets not fixtures, to stay consistent